README.md

```diff
@@ -39,6 +39,7 @@ to the RDO project.
 
 ## Checklist for sending your first RDO gerrit Review:
 
+* [ ] Run `git checkout -b <#easyfix_issue_number>` to create and checkout to a new branch on the issue you are working
```

Comment:
sorry for being picky but It'd be better git checkout -b easyfix/<#easyfix_issue_number>